lib/msal-core/src/Request.ts

```diff
@@ -0,0 +1,33 @@
+import { User } from "./User";
+
+
+/**
+ * TODO:
+ * prompt: We considered making this "enum" in the request instead of string, however it looks like the allowed
+ * list of prompt values kept changing over past couple of years. There are some undocumented prompt values for some
+ * internal partners too, hence the choice of generic "string" type instead of the "enum".
+ *
+ * claimsRequest: This will be a part of the feature "Conditional Access"
+ *
+ * correlationId: No code yet, need to support this as a part of logging sanity check asap
+ *
+ * account: Next PR on request will be user->account code changes
+ *
+ * THIS NOTE IS TEMPORARY FOR TRACTION
+ */
+
+/**
+ *
+ */
+export type AuthenticationParameters = {
```

Comment:
We need to change either this name or AuthenticationRequestParameters.ts - this could potentially be confusing in the future.

Reply:
'AuthenticationParameters' is the name agreed across libraries. Even if we want to change it, it cannot be decided here. I also was thinking on the same lines that we may have to change 'AuthenticationRequestParameters' to 'FinalRequestParameters' or 'ServerRequestParameters' as that file is what the final request to server looks like. Any suggestions?

Reply:
Can we make this change in a separate PR? I will follow up immediately once this is approved.